pubspec.yaml

```diff
@@ -44,7 +44,7 @@ dependencies:
   flutter_animate: ^4.5.0
   flutter_colorpicker: ^1.1.0
   live_activities: ^1.9.3
-  home_widget: ^0.5.0
+  home_widget: ^0.6.0
```

Comment:
Let's keep this at 0.5.0 for now, as upgrading flutter to >=3.20 will cause problems in persistent_bottom_nav_bar. Will upgrade them in next versions